src/styles/index.css

```diff
@@ -0,0 +1,154 @@
+.body {
+  margin: 0;
+}
+
+.button {
+  width: 166px;
+  height: 40px;
+  border-radius: 5px;
+  margin: 16px;
+  background: #00acdc;
+  border: 0;
+  align-items: center;
```

Comment:
The `.button` class has the `align-items` property which has no effect unless the display is set to flex or grid. If this is intended to be a flex container, you should specify `display: flex;`. If not, then this property should be removed.